pkg/policy/policy.go

```diff
@@ -162,8 +166,12 @@ func (policy Policy) Validate(bucketName string) error {
 }
 
 // ParseConfig - parses data in given reader to Policy.
```

Comment:
We can explain `strict` parameter's functionality here, something like:
`'strict' boolean enforces 'version' config information to be set to its default if it doesn't exist.`